recipes/mkl-service/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "2.0.1" %}
+{% set buildnumber = 0 %}
```

Comment:
If a variable is only used once its better not to have it to give the update bot a better chance at parsing it correctly.
```suggestion
```